src/mozanalysis/metrics/desktop.py

```diff
@@ -2,6 +2,8 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from textwrap import dedent
```

Comment:
If you're trying to be tidy about indentation, have you checked what the generated query looks like?

The existing codebase _tries_ to have sensible and consistent indentation in the finally-built query, though this is a bit of a house of cards. If you care about indentation, then please match or improve the pre-existing pattern?

Reply:
I'll check. I'm agnostic about the relative importance of "nice code" and "nice query." Hopefully a human never sees the latter, but if they do, it's probably worth making it intelligible.

Reply:
All right, revised.

Reply:
Given we're metaprogramming SQL, "print the programatically generated query and debug it directly" is going to be a vital development tool when things go wrong adding new metrics and data sources. We want to make this as easy as possible; sensible/consistent indentation makes this slightly less awful.

Rethinking indentation everywhere is definitely on the table, but we should act intentionally.